js/repl/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "babel-repl",
+  "version": "0.0.1",
+  "private": true,
+  "dependencies": {
+    "flow-bin": "^0.52.0",
+    "glamor": "^2.20.39",
+    "lodash.camelcase": "^4.3.0",
+    "lz-string": "^1.4.4",
+    "prettier": "^1.5.3",
+    "react": "^16.0.0-beta.3",
+    "react-dom": "^16.0.0-beta.3",
+    "react-scripts": "1.0.10"
```

Comment:
Maybe merge this with the root `package.json` so we don't have to `yarn install` in two separate directories? I'm not sure what the benefit of splitting them is.